tools/stats/print_test_stats.py

```diff
@@ -780,7 +777,7 @@ def assemble_flaky_test_stats(duplicated_tests_by_file: Dict[str, DuplicatedDict
         for suite_name, testcase_to_runs in suite_to_dict.items():
             for testcase_name, list_of_runs in testcase_to_runs.items():
                 num_green, num_red = process_intentional_test_runs(list_of_runs)
-                if num_green > 0:   # Otherwise, it's likely just a failing test
+                if num_green > 0 and num_red > 0:   # Flaky tests show different results in consecutive reruns
```

Comment:
Gotta add the num_red condition in the case that it's a duplicate test case on purpose. 

Disclaimer: tests that are intentionally spawned several times will be marked flaky if different spawns show differing results. Since that is still problematic, I think it's ok for those to be considered flaky (better safe than miss real flaky tests),